webpack/webpack.dev.babel.js

```diff
@@ -87,8 +87,7 @@ module.exports = Object.assign({}, webpackConfig, {
     new StyleLintPlugin({
       configFile: '.stylelintrc',
       files: '**/*.less',
-      syntax: 'less',
-      failOnError: false
```

Comment:
I think we are saying the same thing. But, it is the default and that is why it is being removed.